README.md

```diff
@@ -132,6 +133,24 @@ Releases to PyPI is an Github Action which needs to be manually triggered.
 [![Release](https://github.com/spetlr-org/spetlr/actions/workflows/release.yml/badge.svg)](https://github.com/spetlr-org/spetlr/actions/workflows/release.yml)
 [![PyPI spetlr badge](https://img.shields.io/pypi/v/spetlr)](https://pypi.org/project/spetlr/)
 
+
+# Requirements and dependencies
+The library has three txt-files at the root of the repo. These files defines three levels of requirements:
+- `requirements_install.txt` - this file contains the required libraries to be able to install spetlr.
+- `test_requirements.txt` - libraries required to run unit- and integration tests
+- `dev_requirements.txt` - libraries required in the development process in order to contribute to the repo
+
+All libraries and their dependencies are added with a fixed version to the configuration file `setup.cfg`.
```

Comment:
This appears to conflict with the above statement about `requirements_intall.txt`, or at least duplicate?